### DISCUSSION TITLE
Edit behaviour inconsistent on Many to One field

### DISCUSSION THREAD

**Discussion by D3VL-Jack** [directus/directus#24905] [Feature Requests]
### Describe the Improvement

When clicking a field, the user expects to be editing the contents of that field. With all other linked collections, this is the case - you click the field and out slides the edit box, with Many To One, it instead slides out the "Select Item" menu (unless you have select disabled). 

While it makes sense from a backend point of view, if select is enabled, technically clicking it and selecting a different item is editing, from an end users perspective this is confusing. 

We've found users are left confused after clicking the large field and a menu with a huge list of items opening instead of being shown the content to edit, them having to exit this and click the much smaller edit target.

![chrome_8apwPDn7N7](https://github.com/directus/directus/assets/21148384/ee0ccb75-b7d9-4ba6-9915-22a1531e2dee)

Proposal:
Swap these two actions, Clicking the larger box should open the edit menu, and have a smaller button to swap the item. Editing is a much more common action than swapping.

**Comment by paescuj** [comment]
To be implemented as per https://discord.com/channels/725371605378924594/1152418116127707228
